Sample/Vungle Sample App/FirstViewController.m

```diff
@@ -221,7 +226,12 @@ - (IBAction)onCheckCurrentStatusButtonTapped:(id)sender {
 }
 
 #pragma mark - VungleSDKDelegate Methods
-
+- (void)vungleRewardUserForPlacementID:(nullable NSString *)placementID {
```

Comment:
For readability - please add spaces after the `#pragma` line, after the `}` on 231 below, and after the `}` on line 234. Always leave one line between methods, and between a pragma statement and the method after.